src/actions/tamanu/psql.rs

```diff
@@ -0,0 +1,140 @@
+use std::path::Path;
+use std::{fs, path::PathBuf};
+
+use clap::Parser;
+use miette::{miette, Context as _, IntoDiagnostic, Result};
+use tracing::{debug, info, instrument};
+
+use crate::actions::Context;
+
+use super::config::{merge_json, package_config};
+use super::{find_tamanu, TamanuArgs};
+
+/// Connect to Tamanu's db via `psql`.
+#[derive(Debug, Clone, Parser)]
+pub struct PsqlArgs {
+	/// Package to look at. By default, the command finds installed version automatically
+	/// If both central and facility server were present, there's no guarantee which server this picks
```

Comment:
```suggestion
	/// Package to load config from.
	///
	/// By default, this command looks for the most recent installed version of Tamanu and tries to
	/// look for an appropriate config. If both central and facility servers are present and
	/// configured, it will pick one arbitrarily.
```

Reply:
The single-line "title" and multiline "body" keeps the short help page (`-h`) looking nice